Cargo.lock

```diff
@@ -1232,7 +1232,7 @@ dependencies = [
 [[package]]
 name = "content-security-policy"
 version = "0.5.4"
-source = "git+https://github.com/servo/rust-content-security-policy/?branch=servo-csp#81f95254fbfe98dd6e130260fd872cf950de9fcd"
+source = "git+https://github.com/servo/rust-content-security-policy/?branch=servo-csp#bd35ef27b7b87c8e7cd05af6e13a2c4b7630165e"
```

Comment:
I am bumping the version of CSP in #36709 as well, which also updates test expectations. Do you mind waiting on that one first and rebase this PR on top of `main` when it merges?

Reply:
Absolutely!

Reply:
The other PR is merged